commerce-theme-minium/commerce-theme-minium-impl/src/main/java/com/liferay/commerce/theme/minium/internal/table/contributor/CommerceAccountClayTableContextContributor.java

```diff
@@ -46,25 +41,16 @@
 	implements ClayTableContextContributor {
```

Comment:
ClayTableHttpContextContributor

Reply:
Http in the middle or at the end? Looks like everything else we have is at the end.